package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/angular/vehicle-inspection-ng.js",
   "scripts": {
     "build": "wfm-template-build -m 'wfm.vehicle-inspection'",
-    "watch": "wfm-template-build -w -m 'wfm.vehicle-inspection'"
+    "watch": "wfm-template-build -w -m 'wfm.vehicle-inspection'",
+    "test": "grunt"
```

Comment:
This popped into my feed, 2 cents to add. No need to add grunt and extra deps, just add eslint to devDeps and maybe do:

```
"test": "eslint lib/*.js lib/**/*.js"
```

Enjoy!